#ifndef MAINWINDOW_H
#define MAINWINDOW_H

#include <QMainWindow>

QT_BEGIN_NAMESPACE
namespace Ui {
class MainWindow;
}
QT_END_NAMESPACE

class MainWindow : public QMainWindow
{
    Q_OBJECT

public:
    MainWindow(QWidget *parent = nullptr);
    ~MainWindow();

private slots:
    void on_actionPop_up_triggered();

    void on_actionExit_triggered();

    void on_button_minus_clicked();

    void on_button_plus_clicked();

private:
    Ui::MainWindow *ui;

    int counter = 1;
    void on_counter_update();
};
#endif // MAINWINDOW_H
